fragments/bastion-ansible.sh

```diff
@@ -56,6 +56,7 @@ function create_metadata_json() {
     "new_nodes": [$new_nodes],
     "deploy_router_or_registry": $deploy_router_or_registry,
     "domainname": "$domainname",
+    "app_subdomain": "${app_subdomain:-"cloudapps.$domainname"}",
```

Comment:
It serves as a fallback default value if app_subdomain is not set by user. There is no reason to make "cloudapps" parametrized - user can just set app_subdomain which is more powerful.